### ISSUE TITLE
Is the custom ordering of notes/folders in a space supposed to be synced to to other devices

### ISSUE THREAD

**Issue by martindkaz** [Make-md/makemd#217]
Is the custom ordering (after drag and drop) of notes/folders in a space supposed to be synced to to other devices over Obsidian Sync? In my case, the order does not sync. Thanks

**Comment by jp-cen** [comment]
Yes, it’s stored in the space database files so if “other file types” is checked, it should sync.

**Comment by martindkaz** [comment]
"Sync all other types" is checked for me on both ends (devices). Actually everything is toggled on in my sync settings. The order is not syncing though

**Comment by jp-cen** [comment]
I'll need to run some tests later this week with multiple devices to see if there are issues with the sync. I'll follow up once I have some answers

**Comment by limex** [comment]
+1
Hi together. I have the same issue. I see that a .mdb file is changed in device A, when I change the order on device A.
But it never goes to device B.

**Comment by acjnn** [comment]
this might still be linked to #443

**Comment by limex** [comment]
I always find it interesting that features that are basic and a must to me, are totally unimportant to obviously the majority of users.  😄 
I guess, I wait another year before I start using make.md 🖖
